docker/compose.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# This is an example/template compose to add eth-duties to your present docker setup.
+# It assumes that you created a docker network called 'ethereum'.
+# Note: I don't know why but for some reason you cannot provide your validator indices
+# or pubkeys on one line via compose. Therefore you need to add each new index and
+# pubkey via a new line (see below).
+
+version: "3.8"
+name: "Compose for eth-duties helper tool"
+services:
+  eth-duties:
+    container_name: "eth-duties"
+    image: "tobiwo/eth-duties:v0.3.0"
```

Comment:
replace 'image' with

build:
      dockerfile: ./docker/dockerfile
      context: ../

this avoids the need to update the tag every time there is a new release